apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerConnectionStringBuilder.cs

```diff
@@ -307,6 +307,20 @@ public SessionPoolManager SessionPoolManager
             set => _sessionPoolManager = GaxPreconditions.CheckNotNull(value, nameof(value));
         }
 
+        /// <summary>
+        /// Specifies whether to allow the connection to check for the presence of the emulator
+        /// environment variable.
+        /// </summary>
+        /// <remarks>
+        /// This property defaults to <see cref="EmulatorDetection.None"/>, meaning that the
+        /// environment variable is ignored.
+        /// </remarks>
+        public EmulatorDetection EmulatorDetection
```

Comment:
I think this should be handled in a fashion much more like properties such as MaximumGrpcChannels, so it becomes part of the connection string itself.

Reply:
All done.